tests/common/macro_support.hpp

```diff
@@ -173,7 +173,7 @@ inline std::vector<Name> sort_names( std::vector<Name>&& names ) {
    { \
       eos::chain::SignedTransaction trx; \
       trx.scope = sort_names( { "eos", #account } ); \
-      trx.emplaceMessage(config::EosContractName, vector<types::AccountPermission>{{#account, "active"}}, \
+      trx.emplaceMessage(config::EosContractName, vector<types::AccountPermission>{}, \
```

Comment:
I removed it because it indicates irrelevant authority included and fails the test.  I thought it was odd, but figured  the question would be answered in this review.  So how do we want to handle this?  Keep this code and write a separate Issue, or should I expand this to fix the check to correctly identify the correct authorities?